doc/DATA_LOADING.md

```diff
@@ -0,0 +1,188 @@
+Data Profiler | Data Loading
+############################
+
+Our object with the data profiler library is a "one function" load of any file. The file should be correctly loaded into a standardized format capable of being profiled by the `Profiler class`. In practice, this means identifying the correct file format, encoding, and then loading the data into a pandas DataFrame (internally). 
+
+Today, it can identify & load any of the following file types:
+
+* A delimited file (CSV, TSV, etc.)
+* JSON object
+* Avro file
+* Parquet file
+* Text file
+
+How to use:
+```python
+import json
+import dataprofiler as dp
+
+# Auto-Detect & Load a file
+data = dp.Data("your_file.csv") 
+
+# Pandas DataFrame
+print(data.data.head(5))
+```
+
+The neat part about the `Data class` is that it is a [factory class](https://www.geeksforgeeks.org/factory-method-python-design-patterns/). What's a factory class? It's a class which lets subclasses decide when to instantiate. Put simply, the `Data class` will load the specific class required to interpret & maniuplate the given file. In way of an example:
+
+```python
+import dataprofiler as dp
+
+# Auto-Detect & Load a file
+data = dp.Data("your_file.csv")
+
+# returns CSVData
+print(type(data))
+
+# <class 'dataprofiler.data_readers.csv_data.CSVData'>
+```
+
+We provide an interface which all data classes must match, with a template provided in the `base_data class` ([data_readers/base_data.py](/data_profiler/data_readers/base_data.py)). What that means is that the end user doesn't have to worry about functions for the specific class, as the template provides a set of required functions. For general purposes, the various the various data classes can be treated the exact same. However, a given class can be checked and special functions added (if desired), making it more extensible. 
```

Comment:
change `For general purposes, the various the various data classes can be treated the exact same` to `For general purposes, the various data classes can be treated the same`